packages/opentelemetry-web/src/utils.ts

```diff
@@ -81,6 +81,12 @@ export function addSpanNetworkEvents(
   addSpanNetworkEvent(span, PTN.REQUEST_START, resource);
   addSpanNetworkEvent(span, PTN.RESPONSE_START, resource);
   addSpanNetworkEvent(span, PTN.RESPONSE_END, resource);
+  if (resource[PTN.ENCODED_BODY_SIZE]) {
+    span.setAttribute(
+      'http.response_content_length',
```

Comment:
Please use the enum here instead of a string

Reply:
Sorry 'bout that, fixed.